core-blocks/gallery/style.scss

```diff
@@ -81,6 +81,13 @@
 		}
 	}
 
+	// Apply max-width to gallery item that contains the add new gallery item button
```

Comment:
This comment could be a bit clearer; it's intent is to make the "add new gallery item" button full-width, so let's just say that 😄:

```css
// Make the "Add new Gallery item" button full-width (so it always appears below other items).
```